.github/workflows/release_canary.yml

```
@@ -41,8 +45,13 @@ jobs:
 
       - name: Create .npmrc
         run: |
+          if [ "$useGPR" = "true" ]; then
+            registry="//npm.pkg.github.com/"
+          else
+            registry="//registry.npmjs.org/"
+          fi
           cat << EOF > "$HOME/.npmrc"
-            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+            $registry:_authToken=$NPM_TOKEN
           EOF
         env:
```

With the change to `useGPR`, you'll probably also need to add the env var..

```suggestion
        env:
          useGPR: ${{ inputs.useGPR }}
```